site/en/docs/extensions/reference/commands/index.md

```diff
@@ -110,3 +107,133 @@ Example:
   ...
 }
 ```
+
+## Examples
+
+The following examples show
+
+### Basic command
+
+Commands allow extensions to map logic to keyboard shortcuts that can be invoked by the user. At
+it's most basic, a command only requires a command declaration in the extension's manifest and a
+listener registration as shown below.
+
+```json
+// manifest.json
+{
+  "name": "Command demo - basic",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "background.js"
+  },
+  "commands": {
+    "inject-script": {
+      "suggested_key": {
+        "default": "Ctrl+Shift+Y",
+        "mac": "Command+Shift+Y"
+      },
+      "description": "Inject a script on the page"
+    }
+  }
+}
+```
+
+```js
+// background.js
+chrome.commands.onCommand.addListener((command) => {
+  console.log(`Command "${command.name}" triggered`);
+});
+```
+
+### Action command
+
+As described in the [Usage][usage] section above, a command can also be mapped to the extension's
```

Comment:
Passive voice. Consider "...you can also map a command to..."

Reply:
Ack